docs-examples/v1.6/quickstart/README.md

```diff
@@ -0,0 +1,51 @@
+# MongoDB Kafka Connector Lite
+
+This docker compose starts the following services:
+
+- MongoDB single node replica set
+- Kafka Connect
+- Kafka Broker
+- Kafka Zookeeper
+
+## Usage
+
+Perform the following actions to install MongoDB Kafka Connector source and sink connectors and send a document through Apache Kafka:
+
+1. Start the docker environment by running the following command:
+
+       docker-compose up -d
+
+2. Enter the a shell in the docker environment with the following command:
+
+       docker exec -it shell /bin/bash
```

Comment:
`Error: No such container: shell`

Reply:
https://jira.mongodb.org/browse/DOCSP-18877

Reply:
Was this command run before the environment was completely built? I believe that would cause this error. There is copy in the tutorial and quick start to address this concern.

As we are currently time constrained getting this up for the Confluent conference tomorrow, I am creating a ticket to document updating the README to be perfect, but at present I think we need to limit ourselves to basic edits for the sake of getting this published on time: https://jira.mongodb.org/browse/DOCSP-18877